### DISCUSSION TITLE
Use Winkel-Tripel projection for countries map

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#174] [Anything Else]
"The Mercator projection currently used in the countries map does not accurately represent the geographic size of countries. Since land mass has some correleation to population of a country, this projection overstates the potential audience size of each country. See https://thetruesize.com/ for a fun interactive example of this distortion.

The Winkel-Tripel is the projection preferred by the National Geographic Society. From a mathematical perspective, it may be the most correctly balanced with respect to the various types of distortion introduced by map projections: area, isotropy (local scaling), flexion (bending), and skewness (lopsidedness).

The Goode homolosine projection would also be a better choice than Mercator, but it splits Greenland in two."

([imported from the old roadmap](https://plausible.nolt.io/146))

**Comment by ghost** [comment]
It looks like you are using [DataMaps](https://datamaps.github.io/). The [default projection is equirectangular](https://github.com/markmarkoh/datamaps#default-options), a more scale-accurate projection than Mercator. Removing this property should immediately improve the map: https://github.com/plausible/analytics/blob/b930b8275250e1fc31ae303d64c548474a3f3936/assets/js/dashboard/stats/countries.js#L73

It'd still be best to use Winkel-Tripel, but I understand that would involve contributing to the DataMaps library.

(Mercator is primarily useful when navigating, but makes countries further away from the equator look more geographically significant than they actually are.)

**Comment by vegeta897** [comment]
Hoping to see this change implemented.
